### PR TITLE
Install Cassandra to /opt/cassandra

### PR DESCRIPTION
**Pull request by cdonati**

### CODE REVIEW

**Comment by cdonati** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/1417/

File: AppController/lib/cassandra_helper.rb [hunk 1]
```diff
@@ -17,14 +17,18 @@
 
 
 # A String that indicates where we install Cassandra on this machine.
-CASSANDRA_DIR = "#{APPSCALE_HOME}/AppDB/cassandra"
+CASSANDRA_DIR = "/opt/cassandra"
```
Comment (menivaitsi) [inline AppController/lib/cassandra_helper.rb]: Is this dir persisted in cloud mode?
I think it should be something like /opt/appscale/cassandra-env to make sure we store in external volumes.

Comment (cdonati) [inline AppController/lib/cassandra_helper.rb]: Do we need to persist the Cassandra binaries? I don't know why it should be different than the other system packages.

Comment (menivaitsi) [inline AppController/lib/cassandra_helper.rb]: @cdonati ok, that makes sense. Thanks!

**Pull request merged**